### DISCUSSION TITLE
Any way to start projects with pages dir instead of app dir?

### DISCUSSION THREAD

**Discussion by pulkitvyas08** [agustinusnathaniel/nextarter-chakra#133] [General]
I see that the project has migrated to app dir but I'm not planning on switching to the app dir and RSC for my production apps. Is there any way to use the template with the older config of pages dir? (apart from the obvious "clone and checkout appropriate commit" solution)

**Comment by agustinusnathaniel** [comment]
Hi @pulkitvyas08 it's really understandable as App Router is considered new 

I put the pages router version in this repo: https://github.com/sozonome/nextarter-chakra-pages-router

Hopefully this helps 🙏

**Reply by pulkitvyas08** [reply]
Amazing thanks!
